config/config_test.go

```diff
@@ -94,7 +94,7 @@ func TestConfig_TrackingDBDir(t *testing.T) {
 	c := config.NewConfig(dir)
 
 	expected := path.Join(dir, "uploads.db")
-	got := c.TrackingDBDir()
+	got := c.CompletedUploadsTrackingDBDir()
```

Comment:
Please add a test for the new resumable uploads DB dir, something similar to:

```
func TestConfig_ResumableUploadsDBDir(t *testing.T) {
	dir := filepath.Join(os.TempDir(), fmt.Sprintf("gphotos-config.%d", time.Now().UnixNano()))
	c := config.NewConfig(dir)

	expected := path.Join(dir, "resumable_uploads.db")
	got := c.ResumableUploadsDBDir()

	if got != expected {
		t.Errorf("Testing get resumable uploads DB dir: expected: %s, got %s", expected, got)
	}

}
```

Reply:
Done.